app/assets/stylesheets/utilities/colours.scss

```diff
@@ -1,6 +1,6 @@
-$colour-black: govuk-colour("black");
-$colour-blue: govuk-colour("blue");
-$colour-light-blue: govuk-colour("light-blue");
-$colour-light-grey: govuk-colour("grey-1");
-$colour-turquoise: govuk-colour("turquoise");
-$colour-white: govuk-colour("white");
+$colour-black: govuk-colour('black');
+$colour-blue: govuk-colour('blue');
+$colour-light-blue: govuk-colour('light-blue');
+$colour-light-grey: govuk-colour('dark-grey');
+$colour-turquoise: govuk-colour('turquoise');
```

Comment:
is this used for a confirmation panel? If so, we should use green instead according to https://github.com/alphagov/govuk-frontend/releases/tag/v3.0.0

Reply:
no I believe we use it for our flash messages, we don't sport any confirmation panel as far as I can tell.